source/slang/slang.natvis

```diff
@@ -842,4 +842,114 @@
     <Type Name="Slang::BasicExpressionType">
         <DisplayString>BasicExpressionType ({*(DeclRefBase*)m_operands.m_buffer[0].values.nodeOperand})</DisplayString>
     </Type>
+
+	<Type Name="Slang::CapabilitySet">
+		<DisplayString>{m_targetSets.map.m_values}</DisplayString>
+		<Expand>
```

Comment:
I changed visual studio today to use tab->spaces, that is probably why

Reply:
Contribution guide is for bare minimum to bootstrap.
And there might be other files using tab-characters.
You just need to watch out for how lines indented and follow the pattern.

Reply:
> Code Style
> Follow our [Coding conventions](https://github.com/shader-slang/slang/blob/master/docs/design/coding-conventions.md) to maintain consistency throughout the project.
> Here are a few highlights
> Indent by four spaces. Don't use tabs except in files that require them (e.g., Makefiles).

This may need to be changed?

Reply:
I don't know natvis can be considered a `code`. This is more of XML document.